staging/src/k8s.io/sample-apiserver/artifacts/example/rbac.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: aggregated-apiserver-clusterrole
 rules:
 - apiGroups: [""]
-  resources: ["namespaces"]
+  resources: ["namespaces", "configmaps"]
```

Comment:
global access to configmaps is not desired. the rolebinding to the `extension-apiserver-authentication-reader` role should cover the read access required